web-server/v0.4/src/components/PageHeader/index.test.js

```diff
@@ -19,4 +37,72 @@ describe('test getBreadcrumb', () => {
     const urlNameList = urlToList('/search').map(url => getBreadcrumb(routerData, url).name);
     expect(urlNameList).toEqual(['search']);
   });
+
+  it('check getBreadcrumb function implementation', () => {
+    const breadcrumbNameMap = { url: { url: 'urlItem' } };
+    const url = 'url';
+    const breadcrumb = breadcrumbNameMap[url];
+    expect(getBreadcrumb(breadcrumbNameMap, url)).toBe(breadcrumb);
+  });
+});
+
+describe('test rendering of TableFilterSelection page component', () => {
+  it('render with empty props', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+  it('Check functions implementation', () => {
```

Comment:
We should consider improving the name of the test to further reflect which function/behavior is being tested.